twcss/prettifycss/index.js

```diff
@@ -0,0 +1,24 @@
+const parse = require('postcss').parse
```

Comment:
why do we have javascript files and not typescript?

Reply:
this plugin is only used for the tailwind css generator to uniformly format the css the same way our typescript files. I don't think postcss accepts typescript modules directly at the moment either and this should not be deployed on the public so just skipped the compile efforts